ingress/controllers/gce/main.go

```diff
@@ -83,6 +75,13 @@ var (
 		`Optional, if this controller is running in a kubernetes cluster, use the
 		 pod secrets for creating a Kubernetes client.`)
 
+	// TODO: Consolidate this flag and running-in-cluster. People already use
+	// the first one to mean "running in dev", unfortunately.
+	useRealCloud = flags.Bool("use-real-cloud", false,
```

Comment:
Meaning "master" is not "in-cluster", because master doesn't run kube-proxy and has a kubectl config and all master pods run with host networking, so in this aspect it's more like a desktop machine than a node.